src/log.ts

```diff
@@ -48,10 +45,13 @@ export interface LogOptions {
   maxEntrySize?: number; // see: https://cloud.google.com/logging/quotas
 }
 
-export type ApiResponse = [Response];
+// tslint:disable-next-line no-any
+export type Metadata = any;
+export type ApiResponse = [Metadata];
 export interface ApiResponseCallback {
-  (err: Error | null, apiResponse?: Response): void;
+  (err: Error | null, apiResponse?: Metadata): void;
```

Comment:
I am not 100% sure what that object is, but I'm 99% sure it ain't going to be a teeny-response.